### DISCUSSION TITLE
Styling of prompt in image-button-response not working.

### DISCUSSION THREAD

**Discussion by nrschoen** [jspsych/jsPsych#1658] [Q&A] [answered]
I am trying to change the font-size of the prompt in an `image-button-response` by adding `stlye="font-size:14px"` to the `<p>` tag. However, the font size remains 18px (see pictures).
```
var faces = [
  { face: 'panicked.jpg', choices: ['jealous', 'panicked', 'arrogant', 'hateful'],
  prompt: '<p stlye="font-size:14px" >jealous: blabla; panicked: blabla; arrogant: blabla; hateful: blabla</p>',
  data: {gender: 'male', correct_response: 1 }},
]; // RMET stimuli
```
![Screen Shot 2021-03-22 at 11 45 26](https://user-images.githubusercontent.com/65179042/111985173-31689680-8b04-11eb-982d-7f201c0ba69b.png)
![Screen Shot 2021-03-22 at 11 46 34](https://user-images.githubusercontent.com/65179042/111985248-4b09de00-8b04-11eb-81bf-20cfb4eed861.png)

**Comment by nrschoen** [comment, marked as answer]
Oh silly me! It was a typo.

**Comment by jodeleeuw** [comment]
You have a typo in your tag, stlye instead of style

**Reply by nrschoen** [reply]
Just saw it! Sorry about that
